### PR TITLE
Create contributing.md

### LINKED ISSUE #3: Add contributing guidelines
* Probably should be a wiki page

### LINKED ISSUE #6: Setup Wiki
The wiki should provide any additional project documentation needed including a list of contributors (students in the group with their associated GitHub usernames), a list of contributions made by each student, and minutes from any group meetings.

### PR DESCRIPTION
**Pull request by skmu104**
closes #3

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Looking to contribute to the project?
+ see below for links and information related to your intended contributions
```
Comment (azizul-i) [inline docs/contributing.md]:
```suggestion
 See below for links and information related to your intended contributions
```

**Review by jwar663 — APPROVED** [review]
Looks good to me!
Is this supposed to be solving issue #3 though?

**Review by dder725 — APPROVED** [review]
Looks awesome @skmu104 !

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Looking to help development?
+-  Please see the [Workflow and git guide](https://github.com/SOFTENG701G1/A1/wiki/Workflow-and-git-guide) documentation to get started
+
+- Look through the [code style](https://github.com/SOFTENG701G1/A1/wiki/Code-Style) documentation so that you can familiarise yourself with the flow and format of the codebase
+
+- If you would like to contact other developers regarding their contributions please see [SPLit team](https://github.com/SOFTENG701G1/A1/wiki/List-of-contributors) 
+
+- Remember to document any api development in [Api documentation](https://github.com/SOFTENG701G1/A1/wiki/API-Description) 
+
+## other useful links
```
Comment (williamlixu) [inline docs/contributing.md]:
```suggestion
## Other useful links
```

**Review by williamlixu — CHANGES_REQUESTED** [review]
Would you please just rename the file CONTRIBUTING.md and also move out from a docs folder into the root directory so it's consistent.
Looks good apart from that :)

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Looking to contribute to the project?
+ see below for links and information related to your intended contributions
+
+## For raising issues and reporting bugs see the links below for further information
+- ###  [Bug reporting](https://github.com/SOFTENG701G1/A1/wiki/Bug-Reporting)
```
Comment (williamlixu) [inline docs/contributing.md]: Save yourself some work and just remove the bug reporting page, issues can cover bugs :)

**Review by williamlixu — APPROVED** [review]
LGTM! Thanks for fixing it!

**Pull request merged**